html/pfappserver/root/static.alt/src/components/pfSearch.vue

```diff
@@ -171,12 +175,18 @@ export default {
     },
     saveSearch () {
       const _this = this
-      this.$store.dispatch('$_nodes/addSavedSearch', {name: this.saveSearchString, query: this.condition}).then(response => {
+      this.$store.dispatch(`$_${this.storeName}/addSavedSearch`, {name: this.saveSearchString, query: this.condition}).then(response => {
         _this.saveSearchString = ''
         _this.showSaveSearchModal = false
       })
     }
   },
+  created () {
+    // Called before the component's created function.
+    if (!this.$options.props.storeName) {
```

Comment:
You could simply [make the prop required](https://vuejs.org/v2/guide/components-props.html#Prop-Validation).